app/views/reports/_call_line.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="report-section">
+  <h2 class="border-bottom">Week of <%= week_range %></h2>
+
+  <h3><%= link_to "DC Call Line", '#', class: 'line-header' %></h3>
```

Comment:
I think I'm missing why we want to put the data into an array first instead of just pulling the data straight from the database. Where does @data get defined?